README.md

```diff
@@ -6,41 +6,40 @@
 [![Try online](https://img.shields.io/badge/try-online-brightgreen)](https://godbolt.org/z/bs11T3)
 [![Documentation](https://img.shields.io/badge/docs-docsforge-blue)](http://flecs.docsforge.com/)
 
-Flecs is a fast and lightweight Entity Component System that lets you build games and simulations with millions of entities ([join the Discord!](https://discord.gg/BEzP5Rgrrp)). Here are some of the framework's highlights:
+Flecs is a fast and lightweight Entity Component System pattern implementation in C that lets you build games and simulations with millions of entities ([join the Discord!](https://discord.gg/BEzP5Rgrrp)). At it's core it uses a [refined approach to building an ECS](https://ajmmertens.medium.com/why-vanilla-ecs-is-not-enough-d7ed4e3bebe5) and as a consequence it is way more flexible than other implementations. Here are some of the framework's highlights:
```

Comment:
I would like to stick to the old description here for these reasons:
- `pattern implementation in C` - it's both C and C++
- `Entity Component System pattern implementation` - `pattern` doesn't add much here
- `and as a consequence it is way more flexible than other implementations` - probably true, and I've seen other ECS implementations have statements like this in their README. I think it's good form though to let the library speak for itself, instead of dunking (a bit) on others.
- `refined approach` - kind of the same as the previous comment, I don't want to imply that flecs is "better" than others, people can decide that for themselves :)

Reply:
~~Let's just say I am doing those changes as an outsider so you're not guilty of dunking on others :P~~

Yeah I see what you mean, agree with most of what you said but I would still want to somehow emphasize that it uses an advancement from a traditional ECS pattern you described in the blog, so I would still want to leave that additional sentence, maybe in some a bit more humble form?

How should we call that approach, since it's exactly an an advancement from the core idea of ECS?